src/app/components/MediaLoader/index.tsx

```diff
@@ -31,25 +35,98 @@ const BumpLoader = () => (
   </Helmet>
 );
 
-const MediaContainer = ({ playerConfig }: { playerConfig: PlayerConfig }) => {
+const AdvertTagLoader = ({ queryString }: { queryString: string }) => {
+  useEffect(() => {
+    // Set window.dotcom to disabled if it doesn't load in 2 seconds.
+    setTimeout(() => {
+      if (window.dotcom.ads.resolves) {
+        window.dotcom.ads.resolves.enabled.forEach(res => res(false));
+        window.dotcom.ads.resolves.getAdTag.forEach(res => res(''));
+      }
+    }, 2000);
+
+    // Initialise the ads object if it hasn't already been loaded.
+    window.dotcom = window.dotcom || { cmd: [] };
+    window.dotcom.ads = window.dotcom.ads || {
+      resolves: {
+        enabled: [],
+        getAdTag: [],
+      },
+      enabled() {
+        return new Promise(resolve => {
+          window.dotcom.ads.resolves.enabled.push(resolve);
+        });
+      },
+      getAdTag() {
+        return new Promise(resolve => {
+          window.dotcom.ads.resolves.getAdTag.push(resolve);
+        });
+      },
+    };
+  }, [queryString]);
+
+  return (
+    <>
```

Comment:
Think this fragment wrapper can be removed along with the `jsxFrag` directive and `React` import.